etl_tests/conftest.py

```diff
@@ -192,3 +212,63 @@ def faiss() -> FAISS:
         index_to_docstore_id={},
         normalize_L2=False,
     )
+
+
+@pytest.fixture(scope="session")
+def anti_recommendation_retrieval_pipeline(
+    faiss: FAISS,
+) -> AntiRecommendationRetrievalPipeline:
+    """Return an AntiRecommendationRetrievalPipeline object."""
+
+    return AntiRecommendationRetrievalPipeline(faiss)
+
+
+@pytest.fixture(scope="session")
+def anti_recommendation_record_key() -> RecordKey:
+    "Return a sample anti-recommendation record key."
+
+    return "Sankoré Madrasah"
+
+
+@pytest.fixture(scope="session")
+def anti_recommendation_article(
+    anti_recommendation_record_key: RecordKey,
+) -> wikipedia.Article:
+    """Return a wikipedia.Article object that will be used as an anti-recommendation."""
+
+    return wikipedia.Article(
+        title=anti_recommendation_record_key,
+        url="https://en.wikipedia.org/wiki/"
+        + anti_recommendation_record_key.replace(" ", "_"),
+        summary="""\
+                 Sankore Madrasah is an ancient center of learning located in Timbuktu, Mali, and is one of
```

Comment:
You can dedent this line to the first column of the file in Python. Then you're not passing a bunch of whitespace to the language model.